frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -255,7 +258,12 @@ function ReaderFooter:resetLayout()
     if new_screen_width == self._saved_screen_width then return end
     local new_screen_height = Screen:getHeight()
 
-    self.progress_bar.width = math.floor(new_screen_width - self.text_width - self.horizontal_margin*2)
+    if self.settings.disable_progress_bar then
+        self.progress_bar.width = 0
+    else
+        self.progress_bar.width = math.floor(
+            new_screen_width - self.text_width - self.horizontal_margin*2)
```

Comment:
it's on purpose to make it easier to read (at least for me). we can also remove all the spaces if you prefer that.